### PR TITLE
Lcs3

### PR DESCRIPTION
**Pull request by chorman0773**

### CODE REVIEW

File: publications/LCS3.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# LCS 3 - LCS Review Committee Members (2019-11-02)
+Publication Date: 2019-11-02
+
+## §1 Forward
```
Comment (rdrpenguin04) [inline publications/LCS3.md]: In this case, it's spelled "Foreword", as in, the word coming before the rest.

**Review by rdrpenguin04 — CHANGES_REQUESTED** [review]

File: publications/LCS3.tex [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+This is a special LCS Document in the same standing as LCS 1 and LCS 2.
+It shall be subjected to annual Committee Review as defined in §2.3 of
+LCS 2.
+
+\subsection{§2 LCS Review Committee
+Members}\label{lcs-review-committee-members}
+
+The following persons have been appointed as members of the LCS Review
+Committee and are voluntarily serving as such: * Connor Horman * Matthew
+Johnson * Maelen Kibler * Ray Redondo
```
Comment (rdrpenguin04) [inline publications/LCS3.tex]: I don't think this got formatted correctly.

**Review by rdrpenguin04 — CHANGES_REQUESTED** [review]

File: publications/LCS3.tex (hunk 2 above)
Comment (chorman0773) [reply publications/LCS3.tex]: You're right. It did not. *Proceeds to yeet pandoc*
I should probably learn LaTex

**Review by rdrpenguin04 — APPROVED** [review]
Looks good now!

**Pull request merged**